src/Stratis.Bitcoin/Consensus/CoinViews/CachedCoinView.cs

```diff
@@ -34,6 +36,31 @@ private class CacheItem
             public TxOut[] OriginalOutputs;
         }
 
+        /// <summary>
+        /// Rewind data with block size that will be kept in the queue until it is persisted to a storage
```

Comment:
it's not block size. 
it kinda depends on the block size but rewind data has less stuff than block so it's always smaller